src/components/forms/KpiAdminForm.vue

```diff
@@ -233,35 +222,26 @@ export default {
     showEditGoalsModal: false,
   }),
 
-  computed: {
-    sheetsEnabled: {
-      get() {
-        // For backwards compatibility, check for any previosly configured sheet
-        // details if the `auto` property doesn't exist on the model.
-        const sheetsEnabled = this.localKpi.auto;
-
-        if (
-          this.kpi &&
-          sheetsEnabled === undefined &&
-          (this.kpi.sheetId || this.kpi.sheetName || this.kpi.sheetCell)
-        ) {
-          return true;
-        }
-        return sheetsEnabled;
-      },
-      set(checked) {
-        this.$set(this.localKpi, 'auto', checked);
-      },
-    },
-  },
-
   watch: {
     kpi: {
       immediate: true,
       handler() {
         if (this.kpi) {
+          // For backwards compatibility, check for any previosly configured sheet
+          // details if the `auto` property doesn't exist on the model.
+          let sheetsEnabled = this.kpi.auto;
+
+          if (sheetsEnabled === undefined) {
+            if (this.kpi.sheetId && this.kpi.sheetName && this.kpi.sheetCell) {
+              sheetsEnabled = true;
+            } else {
+              sheetsEnabled = false;
+            }
```

Comment:
A.K.A. ```sheetsEnabled = this.kpi.sheetId && this.kpi.sheetName && this.kpi.sheetCell;``` om du vil. ;)

Reply:
Sikkert avslørende, men her trenger man faktisk at denne er en bool, ikke en av verdiene? Dette var også for å unngå tidligere uønsket bruk av `!!`.